src/main/java/com/example/onlinebookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -17,7 +17,11 @@ public class UserRegistrationRequestDto {
     @Email
     private String email;
     @Length(min = 4,max = 255)
+    @NotNull
     private String password;
+    @Length(min = 4,max = 255)
+    @NotNull
+    private String repeatPassword;
```

Comment:
read the requirements more carefully
![image](https://github.com/satori11111/Online-Book-Store/assets/48316721/16af8391-aa57-48f0-8494-2bfd2ff63189)